e2e/test/scenarios/question-reproductions/reproductions-1.cy.spec.js

```diff
@@ -213,15 +213,19 @@ function saveQuestion(name) {
 function archiveQuestion(questionName) {
   navigationSidebar().findByText("Our analytics").click();
   openEllipsisMenuFor(questionName);
-  popover().findByText("Move to trash").click();
+  popover().findByText("Archive").click();
 }
 
 function unarchiveQuestion(questionName) {
   navigationSidebar().within(() => {
-    cy.findByText("Trash").click();
+    cy.icon("ellipsis").click();
   });
-  openEllipsisMenuFor(questionName);
-  popover().findByText("Restore").click();
+  popover().findByText("View archive").click();
+  cy.findByText(questionName)
+    .parent()
+    .within(() => {
+      cy.icon("unarchive").click({ force: true });
+    });
```

Comment:
Reverts this
https://github.com/metabase/metabase/commit/7460cad85988b48202de19e0a91463bee205d331#diff-840f1b139269fb00bc74d0fec8a60d0d2ecee89c59379d1546c9492e5f433c7fL74